lib/stdlib/doc/src/uri_string_usage.xml

```diff
@@ -0,0 +1,323 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!DOCTYPE chapter SYSTEM "chapter.dtd">
+
+<chapter>
+  <header>
+    <copyright>
+      <year>2020</year>
+      <year>2020</year>
+      <holder>Ericsson AB. All Rights Reserved.</holder>
+    </copyright>
+    <legalnotice>
+      Licensed under the Apache License, Version 2.0 (the "License");
+      you may not use this file except in compliance with the License.
+      You may obtain a copy of the License at
+
+          http://www.apache.org/licenses/LICENSE-2.0
+
+      Unless required by applicable law or agreed to in writing, software
+      distributed under the License is distributed on an "AS IS" BASIS,
+      WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+      See the License for the specific language governing permissions and
+      limitations under the License.
+
+    </legalnotice>
+
+    <title>Uniform Resource Identifiers</title>
+    <prepared>Péter Dimitrov</prepared>
+    <responsible></responsible>
+    <docno></docno>
+    <approved></approved>
+    <checked></checked>
+    <date>2020-09-30</date>
+    <rev>PA1</rev>
+    <file>uri_string_usage.xml</file>
+  </header>
+  <section>
+    <title>Basics</title>
+    <p>At the time of writing this document there are
+    two major standards concerning Universal Resource Identifiers and
+    Universal Resource Locators:
+    <url href="https://www.ietf.org/rfc/rfc3986.txt">RFC 3986 - Uniform Resource Identifier (URI): Generic Syntax</url> and
+    <url href="https://url.spec.whatwg.org/">WHAT WG URL - Living standard</url>.
+    The former is a classical standard with a proper formal syntax (ABNF) describing
+    the grammar, while the latter is a living document describing the current pratice,
+    that is, how a majority of web browsers work with URIs. WHAT WG URL is web focused
+    and it has no formal grammar but a plain english description of the algorithms
+    that should be followed.</p>
+    <p>You may ask the question, what is the difference between them, if any? The
+    answer is that they provide an overlapping definition for resource indentifiers
+    and they are not compatible. OTP implemented
+    <url href="https://www.ietf.org/rfc/rfc3986.txt">RFC 3986</url> and the term URI will
+    be used throughout this document. A URI is an identifier, a strings of characters
+    that identifies a particular resource.</p>
+    <p>
+    For a more complete problem
+    statement regarding the URIs, please check
+    <url href="https://tools.ietf.org/html/draft-ruby-url-problem-01">URL Problem
+    Statement and Directions</url>.</p>
+
+    <p>Please note that what follows in the coming chapters are based on our
+    interpretation of the standard
+    (<url href="https://www.ietf.org/rfc/rfc3986.txt">RFC 3986</url>).
```

Comment:
You're also implementing at least part of the urlencoded query strings I believe. Manual page links to the HTML 5.2 spec.

Reply:
I opted not to have a full list of standards in the Users' guide so I had removed this paragraph.